# M_mission_loc.msg
# ID 190 Mission_loc

# --- Common --- 
# 发送端 编号 
uint8 Send_ID		
# 接收端 编号
uint8 Receive_ID	 

# --- M_mission_loc ---
# state uint8
uint8 state

# x float
float32 x	
	
# y float
float32 y

# z float
float32 z

# yaw float
float32 yaw


